src/core/scene/a-scene.js

```diff
@@ -49,6 +50,7 @@ module.exports.AScene = registerElement('a-scene', {
     createdCallback: {
       value: function () {
         this.isIOS = isIOS;
```

Comment:
this can be handled in a separate commit, but what do you think about consolidating these booleans to a single `this.supports = {ios: true, mac: true}` object property?

Reply:
Well, the semantics are less `supports` then `is`... I do think the map form will prevent the number of `is` permutations down.

Reply:
I think we should do this one later, there are a bunch of helper functions this might be good for but may want to discuss the approach a bit more